ruff_cli/src/main.rs

```diff
@@ -91,6 +91,12 @@ fn resolve(
 pub fn main() -> Result<ExitCode> {
     // Extract command-line arguments.
     let (cli, overrides) = Cli::parse().partition();
+
+    drop(std::panic::catch_unwind(|| {
```

Comment:
I think you meant to use [`std::panic::set_hook`](https://doc.rust-lang.org/std/panic/fn.set_hook.html)? Maybe also tell user to report the backtrace by re-run with `RUST_BACKTRACE=1`.

Reply:
I meant to use `catch_unwind` because it does not replace the default panic output it just gets invoked before it so the regular output telling the user to re-run with RUST_BACKTRACE=1 is still printed.

Reply:
Ah yes, thanks! Good catch :) I have updated it to use `set_hook` as you suggested and now it only prints when it actually panics. I am still invoking the default hook for the RUST_BACKTRACE=1 message.